<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(37,211,102);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="8" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item row="3" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: White;</string>
          </property>
          <property name="text">
           <string>PREPROCESSING UTILS</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PB_url2tag">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{background: rgb(18, 140, 126);
border: 5px  solid  rgb(255, 255, 255);
border-style: outset;
border-width: 2px;
border-radius: 10px;
color: white;}

QPushButton:hover {
    color: #000;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #fff
        );
    }</string>
          </property>
          <property name="text">
           <string>URL To Tag</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PB_erase_removed_messages">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{background: rgb(18, 140, 126);
border: 5px  solid  rgb(255, 255, 255);
border-style: outset;
border-width: 2px;
border-radius: 10px;
color: white;}

QPushButton:hover {
    color: #000;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #fff
        );
    }</string>
          </property>
          <property name="text">
           <string>Erase Removed Messages</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PB_erase_multimedia">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{background: rgb(18, 140, 126);
border: 5px  solid  rgb(255, 255, 255);
border-style: outset;
border-width: 2px;
border-radius: 10px;
color: white;}

QPushButton:hover {
    color: #000;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #fff
        );
    }</string>
          </property>
          <property name="text">
           <string>Erase Multimedia Messages</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CB_add_context">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: White;</string>
          </property>
          <property name="text">
           <string>Add Context</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="TE_context">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255,255,255);
border: 2px  solid  rgb(255, 255, 255);
border-radius: 20px;
border-color: rgb(0,0,0);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CB_custom_label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: White;</string>
          </property>
          <property name="text">
           <string>Add custom label</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QComboBox" name="COB_save_options">
            <property name="styleSheet">
             <string notr="true">background-color: White;
border-color: Black;
border-radius: 10px;</string>
            </property>
            <item>
             <property name="text">
              <string>Save in .CSV with columns: ['Conversation']</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PB_save_modifications">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{background: rgb(18, 140, 126);
border: 5px  solid  rgb(255, 255, 255);
border-style: outset;
border-width: 2px;
border-radius: 10px;
color: white;}

QPushButton:hover {
    color: #000;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #fff
        );
    }</string>
            </property>
            <property name="text">
             <string>Save Modifications</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="PB_root_folder_select">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{background: rgb(18, 140, 126);
border: 5px  solid  rgb(255, 255, 255);
border-style: outset;
border-width: 2px;
border-radius: 10px;
color: white;}

QPushButton:hover {
    color: #000;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #fff
        );
    }</string>
          </property>
          <property name="text">
           <string>Select Root Directory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="LB_root_directory">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255,255,255);
border: 2px  solid  rgb(255, 255, 255);
border-radius: 10px;
border-color: rgb(0,0,0);</string>
          </property>
          <property name="text">
           <string>Directory will be displayed when selected...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QVBoxLayout" name="VL_utterance">
        <item>
         <widget class="QLabel" name="LB_selected_file">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255,255,255)</string>
          </property>
          <property name="text">
           <string>UTTERANCE FILE: CURRENTLY EMPTY</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="CB_view_completed_files">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">color: White;</string>
          </property>
          <property name="text">
           <string>View Completed Files</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="topMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QLineEdit" name="LE_target_user">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255,255,255);
border: 2px  solid  rgb(255, 255, 255);
border-radius: 10px;
border-color: rgb(0,0,0);</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="CB_highlight_user">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: White;</string>
            </property>
            <property name="text">
             <string>Highlight target user</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="topMargin">
           <number>40</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QPushButton" name="PB_previous_file">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{background: rgb(18, 140, 126);
border: 5px  solid  rgb(255, 255, 255);
border-style: outset;
border-width: 2px;
border-radius: 10px;
color: white;}

QPushButton:hover {
    color: #000;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #fff
        );
    }</string>
            </property>
            <property name="text">
             <string>Previous File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PB_recover_contexts">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{background: rgb(18, 140, 126);
border: 5px  solid  rgb(255, 255, 255);
border-style: outset;
border-width: 2px;
border-radius: 10px;
color: white;}

QPushButton:hover {
    color: #000;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #fff
        );
    }</string>
            </property>
            <property name="text">
             <string>Recover Contexts  From File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PB_next_file">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{background: rgb(18, 140, 126);
border: 5px  solid  rgb(255, 255, 255);
border-style: outset;
border-width: 2px;
border-radius: 10px;
color: white;}

QPushButton:hover {
    color: #000;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #fff
        );
    }</string>
            </property>
            <property name="text">
             <string>Next File</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="WL_utterance">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QListWidget{background-color: rgb(255,255,255);
border: 2px  solid  rgb(255, 255, 255);
border-radius: 20px;
border-color: rgb(0,0,0);
}

</string>
          </property>
          <property name="lineWidth">
           <number>3</number>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragOnly</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::MoveAction</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ContiguousSelection</enum>
          </property>
          <property name="movement">
           <enum>QListView::Free</enum>
          </property>
          <property name="spacing">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>100</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>100</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="PB_save_directory_select">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{background: rgb(18, 140, 126);
border: 5px  solid  rgb(255, 255, 255);
border-style: outset;
border-width: 2px;
border-radius: 10px;
color: white;}

QPushButton:hover {
    color: #000;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #fff
        );
    }</string>
          </property>
          <property name="text">
           <string>Select Save Directory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="LB_save_directory">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255,255,255);
border: 2px  solid  rgb(255, 255, 255);
border-radius: 10px;
border-color: rgb(0,0,0);</string>
          </property>
          <property name="text">
           <string>Directory will be displayed when selected...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>500</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255,255,255)</string>
        </property>
        <property name="text">
         <string>WhatsApp</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LB_title">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255,255,255)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resources/logos.qrc">:/whatsapp_logo/ee994-logo-whatsapp-png.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255,255,255)</string>
        </property>
        <property name="text">
         <string>Utterance Preprocessing</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../resources/logos.qrc"/>
 </resources>
 <connections/>
</ui>
